src/bentoml/_internal/store.py

```diff
@@ -170,14 +170,26 @@ def register(self, tag: t.Union[str, Tag]):
         self._fs.makedirs(item_path)
         try:
             yield self._fs.getsyspath(item_path)
-        finally:
+        except Exception:
+            raise
```

Comment:
What's this for?

Reply:
If any exceptions where that happens during the context manager, we should raise it

Reply:
This can be removed and it works the same